test/tests/test-nonguaranteed-return.bruh

```diff
@@ -0,0 +1,9 @@
+define reverse(number n -> number):
```

Comment:
what is the difference between this test and `test-none-return`? judging from the name, shouldn't the function in `test-none-return` be returning none?

Reply:
`test-none-return` is the same as` test-nonguaranteed-return` right now since they are both caught by nonguaranteed return err

Reply:
word